.mockery.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  github.com/nephio-project/nephio/controllers/pkg/giteaclient:
+    interfaces:
+      GiteaClient:
+        config:
+          dir: "{{.InterfaceDir}}"
```

Comment:
The `{{.InterfaceDir}}` indicates to generate the mock file in the same directory where the interface is located.